simple-dns/src/dns/rdata/caa.rs

```diff
@@ -0,0 +1,79 @@
+use crate::dns::{CharacterString, PacketPart};
+
+use super::RR;
+
+// RFC 8659: Allow domain name holders to indicate whether they are authorized to issue digital certificates for particular domain name
+// Used as a security policy for certificate authorities
+#[derive(Debug, PartialEq, Eq, Hash, Clone)]
+pub struct CAA<'a> {
+    // Critical or noncritical indicator
+    pub flag: u8,
+    // Property described in the VALUE field. One of `issue`, `issuewild`, or `iodef`
+    pub tag: CharacterString<'a>,
+    // Value associated with property tag
+    pub value: CharacterString<'a>,
+}
+
+impl<'a> RR for CAA<'a> {
+    const TYPE_CODE: u16 = 257;
+}
+
+impl<'a> CAA<'a> {
+    /// Transforms the inner data into it's owned type
```

Comment:
>it's [](http://example.com/codeflow?start=39&length=4)

nit: its (yes, I know it's like that in the other files but that doesn't mean THIS one needs to be wrong :p)

Reply:
Thanks for pointing out it is wrong in the other files. I will fix that.

English is not my first language, some errors may occur